index.html

```diff
@@ -0,0 +1,65 @@
+<!doctype html>
+
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+
+  <title>COVID-19 Cases in the United States</title>
+  <meta name="description" content="COVID-19 Cases in the United States">
+  <meta name="author" content="Locksley Kolakowski and Lynzley Kolakowski">
+
+  <link rel="icon" href="images/covidFavicon.ico">
+  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css" integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk" crossorigin="anonymous">
+  <link rel="stylesheet" href="css/style.css">
+
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/fetch/2.0.3/fetch.min.js"></script>
+  <script src="https://d3js.org/d3-dsv.v1.min.js"></script>
+  <script src="https://d3js.org/d3-fetch.v1.min.js"></script>
+
+  <script src="js/interactive.js"></script>
```

Comment:
Your file needs to come after HTML content, otherwise it won't have access to elements when it loads